Shurgeken/Assets/Scripts/Gameplay/AnimationController.cs

```diff
@@ -13,14 +13,18 @@ void Start ()
         data = GetComponent<DataController>();
         animator = GetComponent<Animator>();	
 	}
-	
+
+    public string[] animation_state_names = {"Idle","Run Forward","Run Backwards","Run Left","Run Right","Crouch","Jumping","Falling","Landing", "Damaged", "Death","Attack"};	
```

Comment:
We need to be careful here, "Attack" != "Melee 1". This caused both the player and the AI to be unable to attack because the state didn't exist.